tests/backup/backup_locked_bucket_test.go

```diff
@@ -840,7 +853,7 @@ var _ = Describe("{DeleteLockedBucketUserObjectsFromAdmin}", func() {
 				createRestore := func(backupName string, restoreName string, namespace string) {
 					defer GinkgoRecover()
 					defer wg.Done()
-					customNamespace := fmt.Sprintf("custom-%s-%v", namespace, time.Now().Unix())
+					customNamespace := fmt.Sprintf("custom-%s", namespace)
```

Comment:
Why did we remove the timestamp?

Reply:
The namespace name becomes more than 64 character and causes the case to fail with: 
```


rpc error: code = Unknown desc = failed to create backup location [autogenerated-backup-location-1696056535] CR: rpc error: code = NotFound desc = failed to create backup location CR [autogenerated-backup-location-1696056535]: namespaces "custom-postgres-backup-pxbackuptask-0-09-30-06h47m47s-1696056661" not found
--


<br class="Apple-interchange-newline">
```